mne_bids/tests/data/tiny_bids/sub-01/ses-eeg/eeg/sub-01_ses-eeg_task-rest_eeg.json

```diff
@@ -1,7 +1,7 @@
 {
     "TaskName": "rest",
     "Manufacturer": "Brain Products",
-    "PowerLineFrequency": 60,
+    "PowerLineFrequency": 50.0,
```

Comment:
No idea why this is changing… @sappelhoff could you maybe take a look?

Reply:
no idea why it was 60 before ... 50 is correct (this was recorded in Berlin) and expected based on the code.